docs/source/conf.py

```diff
@@ -91,7 +91,17 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = [
+        'datatypes/**',
+        'developer_guide/**',
+        'get_started/**',
+        'howto/installation_more/index.rst',
+        'import_export/**',
+        'querying/**',
```

Comment:
Isn't this already migrated? The queryhelp and intro files are empty at least and can be deleted I reckon. Also the content of `append.rst` is either commented out or is content that is already addressed in `howto/data.rst` or `topics/database.rst`. So I think we can just remove the whole thing. The other file `backend.rst` I would also just delete. Minimal content and not actually we should recommend to users.

Reply:
I might have missed empty files. In that case, please feel free to just delete them.